app/assets/javascripts/govuk_publishing_components/components/accordion.js

```diff
@@ -106,12 +111,12 @@ window.GOVUK.Modules = window.GOVUK.Modules || {};
     srPause.classList.add('govuk-visually-hidden')
     srPause.innerHTML = ', '
 
-    // Build addtional copy for assistive technology
-    var srAddtionalCopy = document.createElement('span')
-    srAddtionalCopy.classList.add('govuk-visually-hidden')
-    srAddtionalCopy.innerHTML = ' this section'
+    // Build additional copy for assistive technology
+    var srAdditionalCopy = document.createElement('span')
+    srAdditionalCopy.classList.add('govuk-visually-hidden')
+    srAdditionalCopy.innerHTML = ' this section'
```

Comment:
Ideally this text would come from the `en.yml` file so it could be translated. The way to do that is to include it in the template via a data attribute on an element, then read it in using the JS.

Reply:
This _is_ worth thinking about / doing.  

...I'm now wondering if this type of translation functionally exists on the Design System and if not what that might mean for the end goal to remove this from them gem 🤔

Reply:
Do you mind if I issue this? This PR is to fix a fairly critical bug and I'd like to get it out sooner rather than later.

Reply:
Yes, that would be great, thanks.